R/sampling.R

```diff
@@ -1,98 +1,101 @@
 #' Sample conditional variables using the Gaussian copula approach
 #'
-#' @param given_ind Vector. The indices of the features to condition upon.
+#' @param index_mu Vector. The indices of the features to condition upon.
 #' @param p Positive integer. The number of features.
-#' @param Xtest_Gauss_trans Vector with the Gaussian transformed features of the observation whose predictions ought to be explained (test data). Dimension \code{1xp} or \code{px1}.
-#' @param Xtest Matrix, data.frame or data.table with the features of the observation whose predictions ought to be explained (test data). Dimension \code{1xp} or \code{px1}.
+#' @param x_test_gaussian Vector with the Gaussian transformed features of the observation whose
+#' predictions ought to be explained (test data). Dimension \code{1xp} or \code{px1}.
+#' @param x_test Matrix, data.frame or data.table with the features of the observation whose
+#' predictions ought to be explained (test data). Dimension \code{1xp} or \code{px1}.
 #'
 #' @inheritParams global_arguments
 #'
-#' @return data.table with \code{noSamp_MC} (conditional) Gaussian samples
+#' @return data.table with \code{n_samples} (conditional) Gaussian samples
 #'
 #' @keywords internal
 #'
 #' @examples
 #' # TODO: Add simple example
 #'
 #' @author Martin Jullum
-sample_copula <- function(given_ind, noSamp_MC, mu, Sigma, p, Xtest_Gauss_trans, Xtrain, Xtest) {
+sample_copula <- function(index_mu, n_samples, mu, sigma, p, x_test_gaussian, x_train, x_test) {
```

Comment:
What is the intuition behind the name change given_ind -> index_mu ? What is mu? I think given_ind is easy to understand along with dependent_ind.

Reply:
My intuition behind `index_mu` is that it represents indices in `mu` (the other argument). Basically   easier to understand the input without looking at the actual code inside the function.

Reply:
@martinju Changed to `index_given`